packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -38,7 +38,64 @@ const double _kScrollbarCrossAxisMargin = 3.0;
 /// animate from [thickness] and [radius] to [thicknessWhileDragging] and
 /// [radiusWhileDragging], respectively.
 ///
-// TODO(Piinks): Add code sample
+/// {@tool dartpad --template=stateless_widget_scaffold}
+/// This sample shows a [Scrollbar] that executes a fade animation as scrolling occurs.
```

Comment:
```suggestion
/// This sample shows a [CupertinoScrollbar] that executes a fade animation as scrolling occurs.
```

Reply:
Yes I have used the sample code of scrollbar.